app/views/suppliers/orders/_delivery_method.html.erb

```diff
@@ -0,0 +1,18 @@
+<% if @shipping_address %>
+  <h5>Shipping Address:</h5>
+    <p>
+      <%= @shipping_address.line_one %></br>
+      <%= @shipping_address.line_two %></br>
+      <%= @shipping_address.city %></br>
+      <%= @shipping_address.state %></br>
+      <%= @shipping_address.postal_code %></br>
+      <%= @shipping_address.country %>
+    </p>
+<% elsif @coordinates %>
+  <h5>Delivery Coordinates:</h5>
+<% else %>
```

Comment:
I will fill in code for the delivery coordinates once I figure that piece out, probably not until I get to the user side of orders. I'm planning on creating another table for coordinates that only belongs to orders.